### DISCUSSION TITLE
How do i deploy content from the root folder instead of public?

### DISCUSSION THREAD

**Discussion by ghost** [bcomnes/deploy-to-neocities#86] [General]
how do i deploy content from the root folder instead of `public/`?

**Comment by bcomnes** [comment]
No, it's not a good idea. It would end up trying to deploy your .git folder, .github folder and a multitude of other files that should not be deployed to neocities, and will likely trigger API errors from neocities. In order to support this, I would need to maintain an ever growing list of default ignore patterns that would never be quite right. 

Enforcing the deploy from a subfolder that you can customize during deploy avoids the problem completely, and encourages a healthy separation between repo meta and site content (and optionally src content). I realize this may seem unnecessary if you have a 0 build website, but this is the best compromise I can come up with.

**Reply by Kaleidosium** [reply]
Do you have any template on this for zero build sites then? I'm not too familiar with GitHub Actions.

**Reply by bcomnes** [reply]
No but I should probably add one. Check out the list of sites using this in the readme. Im pretty sure most of them are zero build.

**Reply by bcomnes** [reply]
I looked at the list and https://github.com/bechnokid/neocities is a decent example of a 0 build deploy.

**Comment by ghost** [comment]
so in the current version of this site, i used another github action that build a jekyll site and then deploy stuff from an invisible _site folder to neocities

i did a 300 iq play
